pkg/apis/config/v1alpha1/resourceinterpretercustomization_types.go

```diff
@@ -139,11 +139,14 @@ type LocalValueRetention struct {
 	// to the desired specification.
 	//
 	// The script should implement a function as follows:
-	//     luaScript: >
-	//         function Retain(desiredObj, observedObj)
-	//             desiredObj.spec.fieldFoo = observedObj.spec.fieldFoo
-	//             return desiredObj
-	//         end
+	//
+	// ```Lua
```

Comment:
```suggestion
	// ```
``` 
or 
```suggestion
	// ```yaml
```
This is a yaml configuration not `lua`.

Reply:
done